pkg/encrypt/encrypt.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encrypt
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/hex"
+
+	"github.com/pingcap/errors"
+)
+
+var (
+	secretKey, _ = hex.DecodeString("a529b7665997f043a30ac8fadcb51d6aa032c226ab5b7750530b12b8c1a16a48")
```

Comment:
hard code here?

Reply:
yes, currently works like DM, or you need the get this in some way.

Reply:
Since we are using Ansible, we can use Ansible Vault.

The workflow is like:

1. Store sensitive messages in Ansible Vault
1. Use environment variables that are set in playbook to pass the values
1. When deploying, Ansible will ask the operator for password to decrypt the message
1. In our code we read the sensitive values from an environment variable

Reply:
@suzaku @leoppro PTAL 
https://github.com/pingcap/tidb-binlog/pull/857/commits/60a5f622e8d604167e618e1f2a4ec59de5f61c9d
Support set secretkey by env

Reply:
Can we go one step further?

1. We don't do any password en/decryption
1. Introduce several environment variables to pass database passwords, for example, you can use `MYSQL_PWD` to specify the password.

In Linux, only the root and the owner of the process can read the environment variables, so it could be considered safe as long as the system administrator know to keep his/her account safe.

Reply:
Related discussion: https://security.stackexchange.com/questions/83180/secure-database-password-stored-in-an-env-variable

Reply:
setting secretkey by enviornment variable is ok now, and I suggest to file an issue for PM: providing a secure and unified encryption way

Reply:
@suzaku @leoppro PTAL